File: solutions/beecrowd/1012/1012.cs
using System;

class Solution {
    static void Main() {
        double a, b, c, pi = 3.14159;
        string line;
        string[] numbers;

        while ((line = Console.ReadLine()) != null) {
            numbers = line.Split(' ');
            a = Convert.ToDouble(numbers[0]);
            b = Convert.ToDouble(numbers[1]);
            c = Convert.ToDouble(numbers[2]);
            
            Console.WriteLine("TRIANGULO: {0:F3}", a * c / 2);
            Console.WriteLine("CIRCULO: {0:F3}", pi * (c * c));
            Console.WriteLine("TRAPEZIO: {0:F3}", c * ((a + b) / 2));
            Console.WriteLine("QUADRADO: {0:F3}", b * b);
            Console.WriteLine("RETANGULO: {0:F3}", a * b);
        }
    }
}

File: solutions/beecrowd/1012/1012.dart
import 'dart:io';

void main() {
  double a, b, c, pi = 3.14159;

  for (var line = stdin.readLineSync();
      line != null;
      line = stdin.readLineSync()) {
    var numbers = line.split(' ');

    a = double.parse(numbers[0]);
    b = double.parse(numbers[1]);
    c = double.parse(numbers[2]);

    print("TRIANGULO: ${(a * c / 2).toStringAsFixed(3)}");
    print("CIRCULO: ${(pi * (c * c)).toStringAsFixed(3)}");
    print("TRAPEZIO: ${(c * ((a + b) / 2)).toStringAsFixed(3)}");
    print("QUADRADO: ${(b * b).toStringAsFixed(3)}");
    print("RETANGULO: ${(a * b).toStringAsFixed(3)}");
  }
}
